requirements/edx/github.in

```diff
@@ -75,6 +75,11 @@ git+https://github.com/edx/django-rest-framework-oauth.git@0a43e8525f1e3048efe4b
 # This dependency will be removed by the Celery 4 upgrade: https://openedx.atlassian.net/browse/PLAT-1684
 git+https://github.com/edx/django-celery.git@756cb57aad765cb2b0d37372c1855b8f5f37e6b0#egg=django-celery==3.2.1+edx.2
 
+# Forked to add support for python2.7 and to fix predicate inversion
+# Once https://github.com/excitedleigh/bridgekeeper/pull/10 is merged, and we get to python3 and django 2, we can
```

Comment:
Should we create a ticket for this and put it on the revenue squad backlog or something?  
I'm assuming otherwise it may get forgotten

Reply:
Yes, that's a good idea.